is/jsonable.ts

```diff
@@ -0,0 +1,53 @@
+import { type CustomJsonable, isCustomJsonable } from "./custom_jsonable.ts";
+
+/**
+ * Represents a JSON-serializable value.
+ *
+ * See {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#description|Description} of `JSON.stringify()` for more information.
+ */
+export type Jsonable =
+  | string
+  | number
+  | boolean
+  | null
+  | Jsonable[]
+  | { [key: string]: Jsonable }
```

Comment:
The type should be like this because "it does not check array or object properties recursively".

```ts
  | unknown[]
  | { [key: string]: unknown}
```